src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	
+	/**
+	 * Represents a Person's block in the address
+	 */
+	
+	public static final String BLOCK_VALIDATION_REGEX = "\\d+";
+	private String blockNumber;
+	
+	public Block(String block) {
+		this.blockNumber = block;
+	}
+	
+	public String getBlockNumber() {
+		return blockNumber;
+	}
+	
+	public boolean isValidBlock(Block test) {
+		return test.getBlockNumber().matches(BLOCK_VALIDATION_REGEX);
```

Comment:
Good attempt on using regex to validate the block number.

Look up \d \b as as start ;-).